File: tests/libpyre/mpi/sanity.cc
// -*- coding: utf-8 -*-
//
// michael a.g. aïvázis
// california institute of technology
// (c) 1998-2011 all rights reserved
//


// for the build system
#include <portinfo>

// grab the timer objects
#include <mpi.h>

// main program
int main() {
    // initialize
    MPI_Init(0, 0);
    // finalize
    MPI_Finalize();
    // all done
    return 0;
}

// end of file

File: tests/pyre.pkg/timers/sanity.py
#! /usr/bin/env python3
# -*- coding: utf-8 -*-
#
# michael a.g. aïvázis <michael.aivazis@para-sim.com>
# (c) 1998-2023 all rights reserved


def test():
    """
    Sanity test: make sure the pure python timer implementations are accessible
    """
    # access the timer bindings
    from pyre.timers.WallTimer import WallTimer
    from pyre.timers.ProcessTimer import ProcessTimer
    # all done
    return


# main
if __name__ == "__main__":
    # run the test
    test()


# end of file
